hapi/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@hapi/boom": "^9.0.0",
+    "@hapi/boom": "^9.1.3",
     "@hapi/hapi": "^19.1.1",
```

Comment:
is it possible to migrate to hapi v 20.xx?

https://hapi.dev/resources/changelog/

I saw some security vulnerability version warning for hapi

Reply:
see issue #19